### ISSUE TITLE
Integrating desktop width top stories

### ISSUE THREAD

**Issue by andrew-nowak** [bbc/simorgh#3144]
Blocked on bbc/psammead#1761

WSFRONTPGS-129

**Is your feature request related to a problem? Please describe.**
There have been some updated designs for Top Story promos for 600px+ breakpoints.

**Describe the solution you'd like**
Pull in the new version of `psammead-story-promo` which includes new support for desktop width top story promos.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

**Comment by ghost** [comment]
Blocked on https://github.com/bbc/psammead/pull/1930

**Comment by jamesbhobbs** [comment]
This is done

**Comment by jamesbhobbs** [comment]
We've integrated the psammead components and not updated the grid

**Comment by jamesbhobbs** [comment]
Blocked by https://github.com/bbc/simorgh/issues/2313

**Comment by amywalkerdev** [comment]
Closed and done, the grid changes will apply to a further design that we will open a new issue for.
